src/App.tsx

```diff
@@ -1,10 +1,56 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoFilter } from './types/TodoFilter';
+import { TodoList } from './TodoMain/TodoList';
+import { TodoLoadingError } from './TodoLoadingError';
 
-const USER_ID = 0;
+const USER_ID = 10397;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isLoadingError, setIsLoadingError] = useState(false);
+
+  const [filteringBy, setFilteringBy] = useState(TodoFilter.All);
+
+  const loadTodos = useCallback(async () => {
+    setIsLoadingError(false);
+    setIsLoading(true);
+
+    try {
+      const todosFromServer: Todo[] = await getTodos(USER_ID);
+
+      setIsLoading(false);
+      setTodos(todosFromServer);
+    } catch (error) {
+      setIsLoadingError(true);
+    }
+  }, []);
+
+  useEffect(
+    () => {
+      loadTodos();
+    },
+    [],
+  );
```

Comment:
It's a normal formating
```suggestion
  useEffect(() => {
      loadTodos();
    }, []);
```